script.service.ltvpl/resources/skins/Default/skin.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
this file is not needed. skin.xml was deprecated many years ago.

Reply:
removed